File: src/main/java/com/airwaves/airwavesweb/tasks/UserController.java
package com.airwaves.airwavesweb.tasks;

import com.airwaves.airwavesweb.datastore.User;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Date;

@RestController
public class UserController {

    @GetMapping(value = "/clean-users")
    public void clean_users() {
        Date currentTime = new Date();
        for (User user : User.getAll()) {
            if (currentTime.getTime() - user.getUpdated().getTime() < (1000 * 60 * 2)) {
                user.delete();
            }
        }
    }
}

File: src/main/kotlin/com/airwaves/airwavesweb/tasks/UserController.kt
package com.airwaves.airwavesweb.tasks

import com.airwaves.airwavesweb.datastore.User.Companion.all
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import java.util.*

@RestController
class UserController {

    @GetMapping("/clean-users")
    fun clean_users() {
        val currentTime = Date()
        for (user in all) {
            if (currentTime.time - user.updated.time > 1000 * 60 * 2) {
                user.delete()
            }
        }
    }

}